### ISSUE TITLE
Tests are broken

### ISSUE THREAD

**Issue by purbon** [logstash-plugins/logstash-input-udp#9]
From: http://build-eu-00.elastic.co/view/LS%20Plugins/view/LS%20Inputs/job/logstash-plugin-input-udp-unit/9/

```
Bundle complete! 2 Gemfile dependencies, 34 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
Using Accessor#strict_set for specs
Run options: exclude {:redis=>true, :socket=>true, :performance=>true, :couchdb=>true, :elasticsearch=>true, :elasticsearch_secure=>true, :export_cypher=>true, :integration=>true, :windows=>true}
..F

Failures:

  1) LogStash::Inputs::Udp receive should receive events been generated
     Failure/Error: client.send("msg #{i}")
     SocketError:
       send: name or service not known
     # ./spec/support/client.rb:28:in `send'
     # ./spec/inputs/udp_spec.rb:32:in `events'
     # ./spec/inputs/udp_spec.rb:31:in `events'
     # ./spec/spec_helper.rb:12:in `input'
     # ./spec/inputs/udp_spec.rb:30:in `events'
     # ./spec/inputs/udp_spec.rb:42:in `(root)'

Finished in 0.631 seconds (files took 1.47 seconds to load)
3 examples, 1 failure
```

**Comment by jsvd** [comment]
finally found the issue: register doesn't actually make the socket listen on the udp port, that happens during run, so it's possible that the test is sending data to a socket that hasn't been open yet

**Comment by purbon** [comment]
cool, good finding man! thanks a lot for taking this.

**Comment by jsvd** [comment]
fixed by #12
